src/js/jsx/sections/transform/TransformPanel.jsx

```diff
@@ -53,9 +53,8 @@ define(function (require, exports, module) {
 
         shouldComponentUpdate: function (nextProps, nextState) {
             return this.state.referencePoint !== nextState.referencePoint ||
-                this.props.active !== nextProps.active ||
                 this.props.disabled !== nextProps.disabled ||
-                !Immutable.is(this.props.document, nextProps.document);
+                !Immutable.is(this.props.document.layers.selected, nextProps.document.layers.selected);
```

Comment:
This fixes a bug that causes the TransformPanel to render repeatedly while DS is initializing the layers in the background.